content/kapacitor/v1.6/working/api.md

```diff
@@ -295,18 +298,20 @@ Define a new task that is enabled on creation.
 POST /kapacitor/v1/tasks
 {
     "id" : "TASK_ID",
+    "template-id" : "TEMPLATE_ID",
     "type" : "stream",
     "dbrps" : [{"db": "DATABASE_NAME", "rp" : "RP_NAME"}],
     "script" : "stream\n    |from()\n        .measurement('cpu')\n",
     "status" : "enabled"
 }
 ```
 
-Response with task `id` and `link`.
+Response with task `id`, `template-id` and `link`.
 
 ```json
 {
```

Comment:
Is this the complete response for the request? It's not as verbose as the previous example.

Reply:
Good catch, yes there are several inconsistencies. To cover bases for now, updated to Response with task details.